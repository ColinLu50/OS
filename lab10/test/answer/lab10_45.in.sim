994
231
235
226
231
220
