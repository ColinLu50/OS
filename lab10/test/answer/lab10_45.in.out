FCFS
2 -> 233 -> 132 -> 205 -> 178 -> 224 -> 171 -> 169 -> 13 -> 174 -> 30
994
SSTF
2 -> 13 -> 30 -> 132 -> 169 -> 171 -> 174 -> 178 -> 205 -> 224 -> 233
231
SCAN
2 -> 11 -> 233 -> 224 -> 205 -> 178 -> 174 -> 171 -> 169 -> 132 -> 30 -> 13 -> 2 -> 0 -> 13 -> 30 -> 132 -> 169 -> 171 -> 174 -> 178 -> 205 -> 224 -> 233
235
C-SCAN
2 -> 11 -> 233 -> 224 -> 205 -> 178 -> 174 -> 171 -> 169 -> 132 -> 30 -> 13 -> 2 -> 0 -> 237 -> 233 -> 224 -> 205 -> 178 -> 174 -> 171 -> 169 -> 132 -> 30 -> 13
226
LOOK
2 -> 11 -> 233 -> 224 -> 205 -> 178 -> 174 -> 171 -> 169 -> 132 -> 30 -> 13 -> 2 -> 13 -> 30 -> 132 -> 169 -> 171 -> 174 -> 178 -> 205 -> 224 -> 233
231
C-LOOK
2 -> 11 -> 233 -> 224 -> 205 -> 178 -> 174 -> 171 -> 169 -> 132 -> 30 -> 13 -> 2 -> 233 -> 224 -> 205 -> 178 -> 174 -> 171 -> 169 -> 132 -> 30 -> 13
220
